cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,9 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Instance().ReadInstanceData()
```

Comment:
@hferentschik 

>  ~/.minishift/config/<machine name>

Yes this seems fine to me too but agree it creates overhead of cleanup it too. 
A kind of trade off IMO to simplicity of what we are trying to implement.